### DISCUSSION TITLE
Why Tycho is named Tycho?

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-tycho/tycho#1632] [Q&A]
I always wondered where / why the name "Tycho" was chosen is there any historical documents describing how the project came up with the name? Has it something to do with 

https://en.wikipedia.org/wiki/Tycho_(lunar_crater)

??

**Comment by mickaelistria** [comment]
Maybe the oldest resources like the project proposal, the first mailing-threads or the first articles or EclipseCon talks about Tycho can tell.
I'm almost certain that the astronomical stuff you found is related; there was a time when Eclipse community was always trying to refer to stellar objects in their names.

**Reply by laeubi** [reply]
I found the project proposal here: https://www.eclipse.org/proposals/tycho/
